.circleci/config.yml

```diff
@@ -125,6 +109,27 @@ commands:
         default: expect_fail
     steps:
       - run: ./bin/smoke_tests << parameters.use_case >>
+  hold-notification:
+    description: Notify Slack about an awaiting approval job
+    parameters:
+      message:
+        default: "$CIRCLE_USERNAME has a pending approval for $CIRCLE_BRANCH"
+        description: A workflow in CircleCI is awaiting approval.
+        type: string
+      url:
+        default: 'https://circleci.com/workflow-run/${CIRCLE_WORKFLOW_ID}'
+        description: The URL to link back to.
+        type: string
+      webhook:
+        default: '${SLACK_WEBHOOK}'
```

Comment:
What Webhook/channel did you use as a matter of interest?

Reply:
this bit is not behind a context so it just uses the one in the environment-variables for the service